src/handlers/pinata.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  // @ts-expect-error ts-migrate(2305) FIXME: Module '"react-native-dotenv"' has no exported mem... Remove this comment to see the full error message
+  PINATA_API_KEY,
+  // @ts-expect-error ts-migrate(2305) FIXME: Module '"react-native-dotenv"' has no exported mem... Remove this comment to see the full error message
+  PINATA_API_SECRET,
+  // @ts-expect-error ts-migrate(2305) FIXME: Module '"react-native-dotenv"' has no exported mem... Remove this comment to see the full error message
+  PINATA_API_URL,
+  // @ts-expect-error ts-migrate(2305) FIXME: Module '"react-native-dotenv"' has no exported mem... Remove this comment to see the full error message
+  PINATA_GATEWAY_URL,
```

Comment:
ts-ignore should be enough? This is a bit trivial here

Reply:
yeah sounds good, ill remove the jargon, the ts-expect-error stuff was copy.+ pasta from another place we use .env vars